internal/config.go

```diff
@@ -107,13 +107,18 @@ const (
 	OplogReplayOplogApplicationMode = "OPLOG_REPLAY_OPLOG_APPLICATION_MODE"
 	OplogReplayIgnoreErrorCodes     = "OPLOG_REPLAY_IGNORE_ERROR_CODES"
 
-	MysqlDatasourceNameSetting = "WALG_MYSQL_DATASOURCE_NAME"
-	MysqlSslCaSetting          = "WALG_MYSQL_SSL_CA"
-	MysqlBinlogReplayCmd       = "WALG_MYSQL_BINLOG_REPLAY_COMMAND"
-	MysqlBinlogDstSetting      = "WALG_MYSQL_BINLOG_DST"
-	MysqlBackupPrepareCmd      = "WALG_MYSQL_BACKUP_PREPARE_COMMAND"
-	MysqlTakeBinlogsFromMaster = "WALG_MYSQL_TAKE_BINLOGS_FROM_MASTER"
-	MysqlCheckGTIDs            = "WALG_MYSQL_CHECK_GTIDS"
+	MysqlDatasourceNameSetting    = "WALG_MYSQL_DATASOURCE_NAME"
+	MysqlSslCaSetting             = "WALG_MYSQL_SSL_CA"
+	MysqlBinlogReplayCmd          = "WALG_MYSQL_BINLOG_REPLAY_COMMAND"
+	MysqlBinlogDstSetting         = "WALG_MYSQL_BINLOG_DST"
+	MysqlBackupPrepareCmd         = "WALG_MYSQL_BACKUP_PREPARE_COMMAND"
+	MysqlTakeBinlogsFromMaster    = "WALG_MYSQL_TAKE_BINLOGS_FROM_MASTER"
+	MysqlCheckGTIDs               = "WALG_MYSQL_CHECK_GTIDS"
+	MysqlBinlogGetExecutedGTIDCmd = "WALG_MYSQL_BINLOG_GET_EXECUTED_GTID_COMMAND"
```

Comment:
This command needs description in .md file with example (like WALG_MYSQL_BINLOG_REPLAY_COMMAND)